Error when running tests

Running `ember test` on my local gives this error:

```
Cannot find module 'os-locale/' from '/Users/dustin/Archive/ember-materialize-shim/node_modules/yargs/'
Error: Cannot find module 'os-locale/' from '/Users/dustin/Archive/ember-materialize-shim/node_modules/yargs/'
    at Function.module.exports [as sync] (/Users/dustin/Archive/ember-materialize-shim/node_modules/resolve/lib/sync.js:33:11)
    at resolvePkg (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/resolve-pkg.js:20:18)
    at pkg (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/pkg.js:17:20)
    at again (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:18:22)
    at /Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:27:7
    at Array.forEach (native)
    at again (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:26:55)
    at /Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:27:7
    at Array.forEach (native)
    at again (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:26:55)
    at /Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:27:7
    at Array.forEach (native)
    at again (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:26:55)
    at /Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:27:7
    at Array.forEach (native)
    at again (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:26:55)
    at /Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:27:7
    at Array.forEach (native)
    at again (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:26:55)
    at depsFor (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/deps-for.js:29:4)
    at statPathsFor (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/lib/stat-paths-for.js:14:15)
    at hashForDep (/Users/dustin/Archive/ember-materialize-shim/node_modules/hash-for-dep/index.js:15:21)
```

I poked around yargs a bit and haven't found anything obvious.  Will continue to investigate.

For some reason npm is skipping os-locale and y18n even though they are [listed as dependencies](https://github.com/yargs/yargs/blob/v3.27.0/package.json#L17).

```console
npm list | grep yargs

│ │ │ │ │ └─┬ yargs@3.27.0
│   │ └─┬ yargs@3.10.0
    └─┬ yargs@4.8.1
      └─┬ yargs-parser@2.4.1
npm ERR! missing: os-locale@^1.4.0, required by yargs@3.27.0
npm ERR! missing: y18n@^3.2.0, required by yargs@3.27.0
```

--- a/package.json
+++ b/package.json
@@ -39,7 +39,9 @@
     "ember-suave": "4.0.0",
     "ember-sublime": "0.0.9",
     "loader.js": "^4.0.1",
-    "node-sass": "3.4.2"
+    "node-sass": "3.4.2",
+    "os-locale": "^1.4.0",
+    "y18n": "^3.2.1"
   },
   "keywords": [
     "ember-addon",

